### DISCUSSION TITLE
Network Stats Panel/Plugin in Grafana

### DISCUSSION THREAD

**Discussion by atibdialpad** [grafana/grafana#53841] [General]
is there a panel/plugin in Grafana that can take a timeseries like {metric_name=loss, tags=<src_latitutde, src_longitude, dst_latitude, dst_longitude>} and render it in a world map with nodes derived from the lat/long and edges with the metric_value along side it ?

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
